idaes/models_extra/power_generation/flowsheets/test/test_NGFC.py

```diff
@@ -85,66 +82,410 @@ def test_build(m):
+        pytest.approx(1e-8, rel=1e-5)
+    assert iscale.get_scaling_factor(
+        m.fs.cathode_HRSG.control_volume.heat) == \
+        pytest.approx(1e-6, rel=1e-5)
+    assert iscale.get_scaling_factor(
+        m.fs.intercooler_s1.control_volume.heat) == \
+        pytest.approx(1e-6, rel=1e-5)
+    assert iscale.get_scaling_factor(
+        m.fs.intercooler_s2.control_volume.heat) == \
+        pytest.approx(1e-6, rel=1e-5)
+    assert iscale.get_scaling_factor(
+        m.fs.anode_HRSG.control_volume.heat) == \
+        pytest.approx(1e-8, rel=1e-5)
+    assert iscale.get_scaling_factor(
+        m.fs.prereformer.control_volume.heat) == \
+        pytest.approx(1e-6, rel=1e-5)
+    assert iscale.get_scaling_factor(
+        m.fs.reformer.control_volume.heat) == \
+        pytest.approx(1e-6, rel=1e-5)
+    assert iscale.get_scaling_factor(
+        m.fs.reformer_recuperator.shell.heat) == \
+        pytest.approx(1e-6, rel=1e-5)
+    assert iscale.get_scaling_factor(
+        m.fs.reformer_recuperator.tube.heat) == \
+        pytest.approx(1e-6, rel=1e-5)
+
+    # work
+    assert iscale.get_scaling_factor(
+        m.fs.anode_blower.control_volume.work) == \
+        pytest.approx(1e-5, rel=1e-5)
+    assert iscale.get_scaling_factor(
+        m.fs.air_blower.control_volume.work) == \
+        pytest.approx(1e-6, rel=1e-5)
+    assert iscale.get_scaling_factor(
+        m.fs.cathode_blower.control_volume.work) == \
+        pytest.approx(1e-5, rel=1e-5)
+    assert iscale.get_scaling_factor(
+        m.fs.air_compressor_s1.control_volume.work) == \
+        pytest.approx(1e-6, rel=1e-5)
+    assert iscale.get_scaling_factor(
+        m.fs.air_compressor_s2.control_volume.work) == \
+        pytest.approx(1e-6, rel=1e-5)
+    assert iscale.get_scaling_factor(
+        m.fs.cathode_expander.control_volume.work) == \
+        pytest.approx(1e-6, rel=1e-5)
+    assert iscale.get_scaling_factor(
+        m.fs.combustor_expander.control_volume.work) == \
+        pytest.approx(1e-6, rel=1e-5)
+    assert iscale.get_scaling_factor(
+        m.fs.NG_expander.control_volume.work) == \
+        pytest.approx(1e-6, rel=1e-5)
+
+    # reaction extents
+    assert iscale.get_scaling_factor(
+        m.fs.combustor.control_volume.rate_reaction_extent[0, "h2_cmb"]) == \
+        pytest.approx(1e2, rel=1e-5)
+    assert iscale.get_scaling_factor(
+        m.fs.combustor.control_volume.rate_reaction_extent[0, "co_cmb"]) == \
+        pytest.approx(1e2, rel=1e-5)
+    assert iscale.get_scaling_factor(
+        m.fs.combustor.control_volume.rate_reaction_extent[0, "ch4_cmb"]) == \
+        pytest.approx(1e5, rel=1e-5)
 
 
 @pytest.mark.integration
-def test_initialize(m):
-    scale_flowsheet(m)
+def test_initialize_power_island(m):
+    solver.options = {
+        "max_iter": 500,
+        "tol": 1e-7,
+        "bound_push": 1e-12,
+        "linear_solver": "ma57",
+        "ma57_pivtol": 1e-3,
+          }
```

Comment:
I don't think these solver options settings do anything.  The solvers created in initialization only have the option tol=1e-7 (I switched the output to debug to check).